EXAMPLES.md

```diff
@@ -649,6 +651,115 @@ authentication
 </details>
 
 
+## My Account API
+
+> [!NOTE]
+> The My Account API is currently available in [Early Access](https://auth0.com/docs/troubleshoot/product-lifecycle/product-release-stages#early-access). Please reach out to Auth0 support to get it enabled for your tenant.
+
+Use the Auth0 My Account API to manage the current user's account.
+
+To call the My Account API, you need an access token issued specifically for this API, including any required scopes for the operations you want to perform.
```

Comment:
For this, you'd also need to add docs about getting said access token. E.g. https://github.com/auth0/Auth0.swift/blob/master/EXAMPLES.md#api-credentials-ea

Otherwise, devs could assume you have to use the audience slot (on the initial login) for that, which is not possible if they're already using their own audience value. This is precisely the problem that MRRT solves.